server/statistics/hot_peer_cache.go

```diff
@@ -60,6 +60,15 @@ var MinHotThresholds = [RegionStatCount]float64{
 	RegionWriteQueryNum: 32,
 }
 
+// DefaultThresholdsUpdateInterval is the default interval to update thresholds.
+// the refresh interval should be less than store heartbeat interval to keep the next calculate must use the latest threshold.
+var DefaultThresholdsUpdateInterval = StoreHeartBeatReportInterval * 0.8 * time.Second
```

Comment:
How about putting them into the same place with `StoreHeartBeatReportInterval` and 8s directly?

Reply:
It cannot be const, because tests need.